docs: add frontmatter to `coin` pages (#23068)
MystenLabs/sui: docs/content/guides/developer/coin/in-game-token.mdx

---
title: In-Game Currency

> description: The Sui Closed-Loop Token standard can be used to create tokens that can be used as currency within a game application.

Reviewer: ```suggestion
description: Use the Sui Closed-Loop Token standard to create tokens that you can use as currency within a game application.
```
